### PR TITLE
+ Added multiline input markup and style

### PR DESCRIPTION
**Pull request by sroy3**

### CODE REVIEW

**Comment by sroy3** [comment]
screen shot 2016-05-04 at 3 03 17 pm

File: docs/_components/form_controls/01_input.html [hunk 1]
```diff
@@ -46,6 +46,42 @@
...
+    <div class="input-field multiline-field form-group">
+        <ul>
+            <li class="input-wrapper">
+                <input type="text" class="js-new-value-input" required placeholder="">
+                <label>Multiline empty</label>
+                <div class="input-actions">
+                    <span class="js-add-value-button">
+                        <i class="add-action" title="Add a new entry"></i>
+                    </span>
+                </div>
```
Comment (wfortin) [inline docs/_components/form_controls/01_input.html]: That's a lot of elements for a button... can we at least get rid of the span ?

File: scss/controls/input-field.scss [hunk 2]
```diff
@@ -124,4 +124,80 @@
...
+          width: 2px;
+          height: 10px;
+
+          content: '';
+
+          background-color: $medium-blue;
+        }
+      }
+    }
+  }
```
Comment (wfortin) [inline scss/controls/input-field.scss]: Did you test with text overflowing the input? It seems that the position absolute will make the icon go over the text.

File: docs/_components/form_controls/01_input.html (hunk 1 above)
Comment (GermainBergeron) [inline docs/_components/form_controls/01_input.html]: Can you use a button instead? This way it will be focusable with keyboard navigation

Comment (sroy3) [inline docs/_components/form_controls/01_input.html]: @GermainBergeron There is no action linked to it. Clicking on it doesn't do anything. A new line is added only when typing.

Comment (JPLachance) [inline docs/_components/form_controls/01_input.html]: yes it does something. Click on the "Add" icon is focussing on the input.

File: scss/controls/input-field.scss [hunk 3]
```diff
@@ -124,4 +124,84 @@
       }
     }
   }
+
+  // Multiline fields
```
Comment (JPLachance) [inline scss/controls/input-field.scss]: I would split it in it's own file.

File: scss/controls/input-field.scss [hunk 4]
```diff
@@ -124,4 +124,84 @@
...
+  &.multiline-field {
+    li {
+      margin-bottom: 30px;
```
Comment (JPLachance) [inline scss/controls/input-field.scss]: Look's enormous to me! I would check with the UX team to make sure.

File: scss/controls/input-field.scss [hunk 5]
```diff
@@ -124,4 +124,84 @@
...
+
+      label {
+        display: none;
+      }
+
+      &:first-child label {
```
Comment (JPLachance) [inline scss/controls/input-field.scss]: Nice trick!

**Pull request merged**